haskell-src/Concordium/Types/ProtocolLevelTokens/CBOR.hs

```diff
@@ -530,6 +530,11 @@ data TaggableMemo
       CBORMemo {cborMemo :: !Memo}
     deriving (Eq, Show)
 
+-- | Unwrap the 'TaggableMemo' into the inner 'Memo'
+taggableMemoInner :: TaggableMemo -> Memo
+taggableMemoInner UntaggedMemo{..} = untaggedMemo
+taggableMemoInner CBORMemo{..} = cborMemo
+
```

Comment:
I find this irritating, but whatever, it's fine.

Reply:
You rather want `untaggedMemo` in both variants?